### PR TITLE
Added Perigee Keyboard with Default and VIA Keymaps

### PR DESCRIPTION
**Pull request by FreshCourageTake**
## Description
Created a new keyboard called the Lunar. Added a default keymap as well as a VIA keymap.

## Types of Changes
Added new keyboard with 2 keymaps

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebmaker/lunar/config.h [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x33F9
```
Comment (drashna) [inline keyboards/keebmaker/lunar/config.h:23]: This vender ID is in used, and needs to be changed.

https://www.the-sz.com/products/usbid/index.php?v=0x33F9&p=&n=

Above 0x4000 is clear, and the further past that, the better.

File: keyboards/keebmaker/lunar/config.h [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+#    define RGB_MATRIX_MAXIMUM_BRIGHTNESS 150 // limits maximum brightness of LEDs to 150 out of 255. Higher may cause the controller to crash.
+#    define RGB_MATRIX_HUE_STEP 8
+#    define RGB_MATRIX_SAT_STEP 8
+#    define RGB_MATRIX_VAL_STEP 8
+#    define RGB_MATRIX_SPD_STEP 10
+#endif
+
+#ifdef RGBLIGHT_ENABLE
+#    define RGBLED_NUM 60 // Number of LEDs
+#endif
```
Comment (drashna) [inline keyboards/keebmaker/lunar/config.h:64]:
```suggestion
```

File: keyboards/keebmaker/lunar/config.h [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+//#define BACKLIGHT_PIN B7
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+#define RGB_DI_PIN D3
+#define RGBLED_NUM 60 // Number of LEDs
+
+
+#ifdef RGB_MATRIX_ENABLE
```
Comment (drashna) [inline keyboards/keebmaker/lunar/config.h:46]: no animation modes enabled?

File: keyboards/keebmaker/lunar/rev1/rev1.c [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "rev1.h"
+
+void matrix_init_kb(void) {
+  matrix_init_user();
+};
```
Comment (drashna) [inline keyboards/keebmaker/lunar/rev1/rev1.c:20]:
```suggestion
```

File: keyboards/keebmaker/lunar/rev1/rev1.c [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+void suspend_power_down_kb(void) {
+    rgb_matrix_set_suspend_state(true);
+    suspend_power_down_user();
+}
+
+void suspend_wakeup_init_kb(void) {
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
```
Comment (drashna) [inline keyboards/keebmaker/lunar/rev1/rev1.c:52]:
```suggestion
```

File: keyboards/keebmaker/lunar/lunar.h [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+**************************************************/
+#include "rev1.h"
+
+
+/**************************************************
+** Layout macros aliases common to all revisions **
+**                                               **
+**      Name of Alias     Matrix Name:           **
+******* ----------------- ------------------- ****/
+#define LAYOUT_lunar_grid LAYOUT_ortho_5x12
```
Comment (drashna) [inline keyboards/keebmaker/lunar/lunar.h:33]:
```suggestion
```

File: keyboards/keebmaker/lunar/keymaps/via/rules.mk [hunk 7]
```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes 
+RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.
+RGB_MATRIX_ENABLE = no
```
Comment (drashna) [inline keyboards/keebmaker/lunar/keymaps/via/rules.mk:3]: why switch off rgb matrix?

File: keyboards/keebmaker/lunar/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+ * | Tab  |   Q  |   W  |   E  |   R  |   T  |                      |   Y  |   U  |   I  |   O  |   P  | Del  |
+ * |------+------+------+------+------+-------                      -------+------+------+------+------+------|
+ * | Esc  |   A  |   S  |   D  |   F  |   G  |                      |   H  |   J  |   K  |   L  |   ;  |  "   |
+ * |------+------+------+------+------+------|                      |------+------+------+------+------+------|
+ * | Shift|   Z  |   X  |   C  |   V  |   B  |                      |   N  |   M  |   ,  |   .  |   /  |Enter |
+ * |------+------+------+------+------+------+                      +------+------+------+------+------+------|
+ * | Ctrl | Ctrl | Alt  | GUI  |Lower | Spc  |                      | Spc  |Raise | Left | Down |  Up  |Right |
+ * `------------------------------------------                      ------------------------------------------'
+ */
+[_QWERTY] = LAYOUT_lunar_grid(
```
Comment (drashna) [inline keyboards/keebmaker/lunar/keymaps/default/keymap.c:46]:
```suggestion
[_QWERTY] = LAYOUT_ortho_5x12(
```

File: keyboards/keebmaker/lunar/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+ * |   ~  |   !  |   @  |   #  |   $  |   %  |                      |   ^  |   &  |   *  |   (  |   )  | Del  |
+ * |------+------+------+------+------+-------                      -------+------+------+------+------+------|
+ * | Del  |  F1  |  F2  |  F3  |  F4  |  F5  |                      |  F6  |   _  |   +  |   {  |   }  |  |   |
+ * |------+------+------+------+------+------|                      |------+------+------+------+------+------|
+ * |      |  F7  |  F8  |  F9  |  F10 |  F11 |                      |  F12 |ISO ~ |ISO | | Home | End  |      |
+ * |------+------+------+------+------+------+                      +------+------+------+------+------+------|
+ * |      |      |      |      |      |      |                      |      |      | Next | Vol- | Vol+ | Play |
+ * `------------------------------------------                      ------------------------------------------'
+ */
+[_LOWER] = LAYOUT_lunar_grid(
```
Comment (drashna) [inline keyboards/keebmaker/lunar/keymaps/default/keymap.c:67]:
```suggestion
[_LOWER] = LAYOUT_ortho_5x12(
```

File: keyboards/keebmaker/lunar/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+ * |   `  |   1  |   2  |   3  |   4  |   5  |                      |   6  |   7  |   8  |   9  |   0  | Del  |
+ * |------+------+------+------+------+-------                      -------+------+------+------+------+------|
+ * | Del  |  F1  |  F2  |  F3  |  F4  |  F5  |                      |  F6  |   -  |   =  |   [  |   ]  |  \   |
+ * |------+------+------+------+------+------|                      |------+------+------+------+------+------|
+ * |      |  F7  |  F8  |  F9  |  F10 |  F11 |                      |  F12 |ISO # |ISO / | Pg Up| Pg Dn|      |
+ * |------+------+------+------+------+------+                      +------+------+------+------+------+------|
+ * |      |      |      |      |      |      |                      |      |      | Next | Vol- | Vol+ | Play |
+ * `------------------------------------------                      ------------------------------------------'
+ */
+[_RAISE] = LAYOUT_lunar_grid(
```
Comment (drashna) [inline keyboards/keebmaker/lunar/keymaps/default/keymap.c:88]:
```suggestion
[_RAISE] = LAYOUT_ortho_5x12(
```

File: keyboards/keebmaker/lunar/keymaps/via/keymap.c [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+ * | Tab  |   Q  |   W  |   E  |   R  |   T  |                      |   Y  |   U  |   I  |   O  |   P  | Del  |
+ * |------+------+------+------+------+-------                      -------+------+------+------+------+------|
+ * | Esc  |   A  |   S  |   D  |   F  |   G  |                      |   H  |   J  |   K  |   L  |   ;  |  "   |
+ * |------+------+------+------+------+------|                      |------+------+------+------+------+------|
+ * | Shift|   Z  |   X  |   C  |   V  |   B  |                      |   N  |   M  |   ,  |   .  |   /  |Enter |
+ * |------+------+------+------+------+------+                      +------+------+------+------+------+------|
+ * | Ctrl | Ctrl | Alt  | GUI  | One  | Spc  |                      | Spc  |  Two | Left | Down |  Up  |Right |
+ * `------------------------------------------                      ------------------------------------------'
+ */
+[_QWERTY] = LAYOUT_lunar_grid(
```
Comment (drashna) [inline keyboards/keebmaker/lunar/keymaps/via/keymap.c:48]:
```suggestion
[_QWERTY] = LAYOUT_ortho_5x12(
```

File: keyboards/keebmaker/lunar/keymaps/via/keymap.c [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+ * |   ~  |   !  |   @  |   #  |   $  |   %  |                      |   ^  |   &  |   *  |   (  |   )  | Del  |
+ * |------+------+------+------+------+-------                      -------+------+------+------+------+------|
+ * | Del  |  F1  |  F2  |  F3  |  F4  |  F5  |                      |  F6  |   _  |   +  |   {  |   }  |  |   |
+ * |------+------+------+------+------+------|                      |------+------+------+------+------+------|
+ * |      |  F7  |  F8  |  F9  |  F10 |  F11 |                      |  F12 |ISO ~ |ISO | | Home | End  |      |
+ * |------+------+------+------+------+------+                      +------+------+------+------+------+------|
+ * |      |      |      |      |      |      |                      |      |      | Next | Vol- | Vol+ | Play |
+ * `------------------------------------------                      ------------------------------------------'
+ */
+[_ONE] = LAYOUT_lunar_grid(
```
Comment (drashna) [inline keyboards/keebmaker/lunar/keymaps/via/keymap.c:69]:
```suggestion
[_ONE] = LAYOUT_ortho_5x12(
```

File: keyboards/keebmaker/lunar/keymaps/via/keymap.c [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+ * |   `  |   1  |   2  |   3  |   4  |   5  |                      |   6  |   7  |   8  |   9  |   0  | Del  |
+ * |------+------+------+------+------+-------                      -------+------+------+------+------+------|
+ * | Del  |  F1  |  F2  |  F3  |  F4  |  F5  |                      |  F6  |   -  |   =  |   [  |   ]  |  \   |
+ * |------+------+------+------+------+------|                      |------+------+------+------+------+------|
+ * |      |  F7  |  F8  |  F9  |  F10 |  F11 |                      |  F12 |ISO # |ISO / | Pg Up| Pg Dn|      |
+ * |------+------+------+------+------+------+                      +------+------+------+------+------+------|
+ * |      |      |      |      |      |      |                      |      |      | Next | Vol- | Vol+ | Play |
+ * `------------------------------------------                      ------------------------------------------'
+ */
+[_TWO] = LAYOUT_lunar_grid(
```
Comment (drashna) [inline keyboards/keebmaker/lunar/keymaps/via/keymap.c:90]:
```suggestion
[_TWO] = LAYOUT_ortho_5x12(
```

File: keyboards/keebmaker/lunar/keymaps/via/keymap.c [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+ * |      |      |      |      |      |      |                      |      |      |      |      |      |      |
+ * |------+------+------+------+------+-------                      -------+------+------+------+------+------|
+ * |      |      |      |      |      |      |                      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|                      |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |                      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+                      +------+------+------+------+------+------|
+ * |      |      |      |      |      |      |                      |      |      |      |      |      |      |
+ * `------------------------------------------                      ------------------------------------------'
+ */
+[_THREE] = LAYOUT_lunar_grid(
```
Comment (drashna) [inline keyboards/keebmaker/lunar/keymaps/via/keymap.c:111]:
```suggestion
[_THREE] = LAYOUT_ortho_5x12(
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
Also, remove the lib/ugfx folder.

**Comment by FreshCourageTake** [comment]
Enough updates were required that it seemed prudent to simply create a new PR.

**Pull request closed without merging**